src/datatypes/interval.coffee

```diff
@@ -165,6 +165,11 @@ module.exports.Interval = class Interval
     else # ol is null
       null
 
+  sameAs: (other, precision) ->
+    if (@ == null or other == null) then return null
+    if (@low == null or @high == null or other.low == null or other.high == null) then return null
+    @low.sameAs(other.low, precision) and @high.sameAs(other.high, precision)
```

Comment:
Would this work for intervals of numbers?  I think the intent of `Same As` is that it works on _all_ intervals (not just DateTimes).  I think this would fail on `Interval[0, 5] same as `Interval[0, 5]`.